tools/ci_build/github/azure-pipelines/templates/download-deps.yml

```diff
@@ -11,7 +11,7 @@ steps:
       packageType: upack
       feed: '/7424c8e4-5c62-490e-95c4-79446f31017c'
       definition: '517c4f6f-5437-4392-a70d-4f15ec5be2f0'
-      version: 1.0.53
+      version: 1.0.58
```

Comment:
>58 [](http://example.com/codeflow?start=19&length=2)

Why the version jumps so much?

Reply:
it bumped from 1.0.56 to 1.0.59 because someone else was also updating this part.